src/app/layout.tsx

```diff
@@ -2,8 +2,9 @@ import type { Metadata } from "next";
 import "../global.css";
 
 export const metadata: Metadata = {
-  title: "Next.js Template",
-  description: "Template for Next.js apps",
+  title: "Ryan Tax Firm",
+  description:
+    "Ryan is a global tax services, software, and technology firm providing an integrated suite of federal, state, local, and international tax services to companies across the world.",
```

Comment:
We definitely want to be pulling these from the metadata objects in Contentful

Reply:
Removed metadata from the layout component so the SEOMetadata component can handle that